server/src/main/java/fr/uga/l3miage/tp4/models/ExaminerEntity.java

```diff
@@ -0,0 +1,20 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import java.util.Set;
+
+@Entity
+public class ExaminerEntity extends UserEntity{
+
+    @Column(length = 6)
+    private String login;
+    private String password;
+    @ManyToOne
+    private TestCenterEntity testCenter;
+
+    @OneToMany(mappedBy = "examiner")
+    private Set<CandidateEvaluationGridEntity> candidateEvaluationGrids;
```

Comment:
cette relation tu la traite de manière unidirectionnelle donc tu ne peux pas mettre de `mappedBy`
```suggestion
    @OneToMany
    @JoinColumn(name="grid_id", referencedColumnName="sheet_number")
    private Set<CandidateEvaluationGridEntity> candidateEvaluationGrids;
```